modules/base/rendering/renderablemodel.h

```diff
@@ -70,9 +71,22 @@ class RenderableModel : public Renderable {
     static documentation::Documentation Documentation();
 
 private:
+    enum class AnimationMode {
+        Once = 0,
+        LoopFromStart,
+        LoopInfinitely,
+        BounceFromStart,
+        BounceInfinitely
+    };
+
     std::unique_ptr<ghoul::modelgeometry::ModelGeometry> _geometry;
+    DistanceUnit _modelScale;
+    glm::dvec3 _scaleVector = glm::dvec3(1.0, 1.0, 1.0);
     bool _forceRenderInvisible = false;
     bool _notifyInvisibleDropped = true;
+    std::string _animationStart = "";
```

Comment:
No need to initialize a `std::string` with `""` as the default constructor of `std::string` does that already